spec/features/user_login_logout_spec.rb

```diff
@@ -0,0 +1,44 @@
+require "rails_helper"
+require "./spec/support/user_login.rb"
+
+RSpec.describe "User login and logout" do
+  let!(:user) { User.create(name: "Markus",
```

Comment:
Avoid using {...} for multi-line blocks.<br>Block body expression is on the same line as the block start.